src/data_structures/bit_tree.rs

```diff
@@ -72,15 +77,15 @@ impl<T: Ord + Default + Copy, Op: PrefixOp<T>> FenwickTree<T, Op> {
         sum
     }
 
-    /// Set the value val at position idx. val will
+    /// Set the value val at position idx. In max bit trees, val will
```

Comment:
```suggestion
    /// Set the value `val` at position `idx`; `val` will
```
and so on.

Reply:
Thanks! The repeating "max bit tree"-bits comes from the differentiation between prefix-max and prefix-sum (which is the more traditional Fenwick-Tree in literature) trees. Prefix-sum trees seem to lack tests, documentation and - if I'm correct - a small implementation detail.
 
I'll just update the docs and then do another PR for the prefix-sum tree.